baseline/pytorch/classify/model.py

```diff
@@ -165,6 +165,18 @@ def init_output(self, input_dim, output_dim, **kwargs):
         return Dense(input_dim, output_dim, activation=kwargs.get('output_activation', 'log_softmax'))
 
 
+class MultiLabelMixin(ClassifierModelBase):
```

Comment:
this seems reasonable.  OTOH, the way I did this seems like it has minimal changes to overall class structure: https://github.com/dpressel/mead-baseline/compare/feature/v2...feature/multilabel

Reply:
My thought was that we have all these hooks in baseline so we should be using them like this when we can. I can look into the more conditional version like your branch.

What was the thought on how we would trigger multilabel? It is just something the reader detects and that percolates up to the model? Or is there something where we can specify the loss and this loss tells us we are multilabel? or just a multilabel flag?

Reply:
Yeah, definitely could handle multiple ways.  The reader could attempt to detect from the labels being comma-separated e.g. that its multi-label, and maybe the model could figure it out from the shape of Y as you mentioned.  My concern was that bad interpretation + implicit behavior would be more likely to lead to an upsetting outcome for the user than just explicitly requesting it.  My 2c, is the user knows if they are trying to build a multilabel model, so why not just make them define that explicitly.  But even so its a valid question which block this makes sense in.

Reply:
One consideration of something like a `is_multilabel` flag is it seems to moving away from the model just being a box that is Tensor-In Tensor-Out which is how we have been trying to move towards.